modules/fedimint-ln-client/src/db.rs

```diff
@@ -12,9 +11,10 @@ use crate::OutgoingLightningPayment;
 #[repr(u8)]
 #[derive(Clone, EnumIter, Debug)]
 pub enum DbKeyPrefix {
-    LightningGateway = 0x28,
     PaymentResult = 0x29,
     MetaOverrides = 0x30,
+    // Lightning Gateway cache records are stored the same way as the server, using 0x45
+    // [`fedimint_ln_common::db::DbKeyPrefix::LightningGateway`].
```

Comment:
It was mainly to re-use the existing code, since we already had a struct definition that we needed. I understand the concern though, we should probably move `db.rs` in `fedimint-ln-common` into the server and only use the db records from within the respective crates.

Reply:
Fixed here: https://github.com/fedimint/fedimint/pull/4567